### PR TITLE
LPS-183356 - Use 'auto' symbols for cropperjs to properly import via esm

### PR DESCRIPTION
**Pull request by bryceosterhaus**
https://issues.liferay.com/browse/LPS-183356

Fixed config that exports cropperjs

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:test:item-selector

**Comment by bryceosterhaus** [comment]
ci:test:item-selector

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
Not sure why 1500 tests ran on ci:test:item-selector... :man_shrugging:

File: modules/apps/frontend-js/frontend-js-dependencies-web/npmscripts.config.js [hunk 1]
```diff
@@ -35,6 +34,7 @@ module.exports = {
 			'graphiql/graphiql.css',
 			{name: 'axe-core', symbols: 'auto'},
 			{name: 'clipboard', symbols: 'auto'},
+			{name: 'cropperjs', symbols: 'auto'},
```
Comment (izaera) [inline modules/apps/frontend-js/frontend-js-dependencies-web/npmscripts.config.js:37]: LGTM as long as it works :sweat_smile:

**Review by izaera — APPROVED** [review]

**Comment by manoelcyreno** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:forward

**Pull request closed without merging**